appveyor.yml

```diff
@@ -109,6 +111,58 @@ build_script:
  - 7z a -tzip -r ..\output\symbols.zip *.dl_ *.ex_ *.pd_
  - cd ..
 
+before_test:
+ - py -m pip install robotframework
+ - py -m pip install robotremoteserver

```

Comment:
Is there a particular reason why this installed twice, also after running submodule update?

Reply:
Perhaps a mistake, for future reference, the other is on [line 65 ](https://github.com/nvaccess/nvda/blob/25f18979960dba1dd1f86b3cf4b34fcacc906b38/appveyor.yml#L65)

Reply:
Ah yes, I remember now. For a little while we were packaging robotRemoteServer with the install of NVDA so that it could be used by the addon. Now we package it within the addon. I will remove this line.